opensrs/xcp.py

```diff
@@ -76,7 +78,7 @@ def set_data(self, data, base_node=None, wrap_scalar=None):
     def to_string(self):
         xmlheader = '<?xml version="1.0" encoding="UTF-8" standalone="no" ?>\n'
         xmlheader += '<!DOCTYPE OPS_envelope SYSTEM "ops.dtd">\n'
-        return xmlheader + ET.tostring(self.message_root)
+        return xmlheader + ET.tostring(self.message_root).decode('utf-8')
```

Comment:
So, OPSMessage deals in unicode, and XCPMessage deals in bytes? I suppose that works.